android/authentication.md

```diff
@@ -1,1769 +1,3 @@
 ---
-title: Authentication
+redirect_to: https://docs.amplify.aws/lib/auth/getting-started?platform=android
```

Comment:
How would you redirect from here?

Reply:
Pretty sure https://aws-amplify.github.io/docs/sdk/android/authentication -> https://docs.amplify.aws/lib/auth/getting-started?platform=android right?